### PR TITLE
Added move-an-element.js

### PR DESCRIPTION
**Pull request by AndriiChuprina**

### CODE REVIEW

File: js-arrays/moveanelement.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    displayArrayMoveAfter();
+}
+function fromPositionToAnother(fromPosition,toPosition) {
+    if(fromPosition<0) {
+        fromPosition = arrayBefore.length - Math.abs(fromPosition);
+    }
+    if(toPosition<0) {
+        toPosition = arrayBefore.length - Math.abs(toPosition);
+    }
+    console.log(fromPosition,toPosition);
```
Comment (dmitry-darly-solutions) [inline js-arrays/moveanelement.js]: remove console

File: js-arrays/moveanelement.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    element = arrayBefore[fromPosition];
+    arrayAfter = arrayBefore.slice();
+    arrayAfter.splice(fromPosition,1);
+    arrayAfter.splice(toPosition,0,element);
+    console.log(element);
```
Comment (dmitry-darly-solutions) [inline js-arrays/moveanelement.js]: remove console

File: js-arrays/moveanelement.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+let arrayBefore = [], arrayAfter = [], arrayFromTo = [] ;
+let fromPos, toPos, element;
+function addElementToArrayMove() {
+    arrayBefore[moveCounter] = document.getElementById('element-move').value;
+    displayArrayMoveBefore();
+    moveCounter++;
+}
+function moveElement() {
+    arrayFromTo = document.getElementById('element-move').value;
+    arrayFromTo = (arrayFromTo.toString()).split(',');
```
Comment (dmitry-darly-solutions) [inline js-arrays/moveanelement.js]: arrayFromTo = document.getElementById('element-move').value.toString().split(',');
not sure if you need toString() for element-move (if this element is input)

Comment (dmitry-darly-solutions) [reply js-arrays/moveanelement.js]: please add check for correct input and show warning

File: js-arrays/moveanelement.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+let moveCounter = 0;
+let arrayBefore = [], arrayAfter = [], arrayFromTo = [] ;
+let fromPos, toPos, element;
```
Comment (dmitry-darly-solutions) [inline js-arrays/moveanelement.js]: not sure if you need this variables as common

File: js-arrays/moveanelement.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    arrayBefore[moveCounter] = document.getElementById('element-move').value;
+    displayArrayMoveBefore();
+    moveCounter++;
+}
+function moveElement() {
+    arrayFromTo = document.getElementById('element-move').value;
+    arrayFromTo = (arrayFromTo.toString()).split(',');
+    fromPos = arrayFromTo[0];
+    toPos = arrayFromTo[1];
+    fromPositionToAnother(fromPos,toPos);
```
Comment (dmitry-darly-solutions) [inline js-arrays/moveanelement.js]: you can pass variables as is
fromPositionToAnother(arrayFromTo[0],arrayFromTo[1]);

File: js-arrays/move-an-element.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+const arrayBefore = [];
+let arrayAfter = [], arrayFromTo = [];
+function addElementToArrayMove() {
+    arrayBefore.push(document.getElementById('element-move').value);
+    displayArrayMoveBefore();
+}
+function moveElement() {
+    arrayFromTo = (document.getElementById('element-move').value).split(',');
```
Comment (dmitry-darly-solutions) [inline js-arrays/move-an-element.js]: declare arrayFromTo  in this function

File: js-arrays/move-an-element.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+function addElementToArrayMove() {
+    arrayBefore.push(document.getElementById('element-move').value);
+    displayArrayMoveBefore();
+}
+function moveElement() {
+    const arrayFromTo = (document.getElementById('element-move').value).split(',');
+    const fromPosition = arrayFromTo[0];
+    const toPosition = arrayFromTo[1];
+    if(isNaN(fromPosition)||isNaN(toPosition)) {
+        return alert(fromPosition + " or " + toPosition+ " is not a number");
```
Comment (dmitry-darly-solutions) [inline js-arrays/move-an-element.js]: use `${fromPosition} or ${toPosition} is not a number` instead

Comment (AndriiChuprina) [reply js-arrays/move-an-element.js]: fixed

File: js-arrays/move-an-element.js (hunk 6 above)
Comment (AndriiChuprina) [reply js-arrays/move-an-element.js]: fixed

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**